manager/src/main/java/com/redhat/service/bridge/manager/api/user/ProcessorsAPI.java

```diff
@@ -110,12 +111,31 @@ public Response listProcessors(@NotEmpty @PathParam("bridgeId") String bridgeId,
     @Operation(summary = "Create a Processor of a Bridge instance", description = "Create a Processor of a Bridge instance for the authenticated user.")
     @POST
     @Path("{bridgeId}/processors")
-    public Response addProcessorToBridge(@PathParam("bridgeId") @NotEmpty String bridgeId, @ValidActionParams @ValidTransformationTemplate @Valid ProcessorRequest processorRequest) {
+    public Response addProcessorToBridge(@NotEmpty @PathParam("bridgeId") String bridgeId, @ValidActionParams @ValidTransformationTemplate @Valid ProcessorRequest processorRequest) {
         String customerId = identityResolver.resolve(jwt);
         Processor processor = processorService.createProcessor(bridgeId, customerId, processorRequest);
         return Response.accepted(processorService.toResponse(processor)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Accepted.", responseCode = "202",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = ProcessorResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403"),
+            @APIResponse(description = "Not found.", responseCode = "404", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Internal error.", responseCode = "500", content = @Content(mediaType = MediaType.APPLICATION_JSON))
+    })
+    @Operation(summary = "Update a Processor instance", description = "Update a Processor instance for the authenticated user.")
```

Comment:
I'd make more explicit that it is possible to update only the filters atm

Reply:
> I'd make more explicit that it is possible to update only the filters atm

OK. I'll make the change.

Reply:
> > Looking at kafka they have `created_at` as well as `updated_at`. Same for connectors, but they have `created_at` and `modified_at`.
> 
> ... and we have `submittedAt` and `modifiedAt`. What a cosmopolitan world in which we live.
> 
> I am happy to bring some consistency to the world.. but not in this JIRA/PR. The change is part of our `ManagedResource` and `Worker` mechanism... So I'd prefer to do as a new JIRA. WDYT?

Sure thing, my comment was just an headsup :) And you don't actually have to do it, I'll create a jira and somebody will do it